### PR TITLE
Add org.regular_dev.biplanes_revival

### PR DESCRIPTION
**Pull request by Casqade**
### Please confirm your submission meets all the criteria

- [X] Please describe your application briefly.
- An old cellphone arcade recreated for PC
- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I have [built][build] and tested the submission locally.
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [X] The domain used for the application ID is controlled by the application developers either directly or through the code hosting (e.g. GitHub, GitLab, SourceForge, etc.). The [application id guidelines][app-id] are followed.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission
[app-id]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id

### CODE REVIEW

File: org.regular_dev.biplanes_revival.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      - cmake --build .
+    builddir: true
+    sources:
+      - type: git
+        commit: 0d5272c014128283a4b9f322a9bbe16fa6e7303e
+        url: https://github.com/regular-dev/biplanes-revival
+    cleanup:
+      - '/include'
+      - '/bin/assets'
+  - name: BiplanesRevival-postinstall
```
Comment (hfiguiere) [inline org.regular_dev.biplanes_revival.yml:24]: this doesn't need to be a separate module

File: org.regular_dev.biplanes_revival.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+      - install -Dm644 flatpak-data/${FLATPAK_ID}.desktop -t /app/share/applications
+      - install -Dm644 flatpak-data/${FLATPAK_ID}.metainfo.xml -t /app/share/metainfo
+      - install -Dm644 flatpak-data/${FLATPAK_ID}.svg -t /app/share/icons/hicolor/scalable/apps
+      - mkdir -p /app/share/biplanes-revival
+      - cp -R assets /app/share/biplanes-revival
+    sources:
+      - type: git
+        commit: 0d5272c014128283a4b9f322a9bbe16fa6e7303e
+        url: https://github.com/regular-dev/biplanes-revival
+finish-args:
```
Comment (hfiguiere) [inline org.regular_dev.biplanes_revival.yml:36]: finish args should be above modules.

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["x86_64"]
```
Comment (hfiguiere) [inline flathub.json:2]: why?

File: org.regular_dev.biplanes_revival.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+id: org.regular_dev.biplanes_revival
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: BiplanesRevival
+build-options:
+  strip: true
+  no-debuginfo: true
```
Comment (hfiguiere) [inline org.regular_dev.biplanes_revival.yml:8]:
```suggestion
```

File: flathub.json (hunk 3 above)
Comment (Casqade) [reply flathub.json:2]: I don't have any AArch64 devices to test the app

Comment (Casqade) [reply flathub.json:2]: And this game is Desktop-only

Comment (hfiguiere) [reply flathub.json:2]: neither is a valid excuse. There are aarch64 desktops. And it surely can't be tested if it is not built.

File: org.regular_dev.biplanes_revival.yml [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  - --socket=pulseaudio
+  - --device=dri
+  - --env=BIPLANES_ASSETS_ROOT=/app/share/biplanes-revival/
+modules:
+  - name: BiplanesRevival
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    build-commands:
+      - ninja
```
Comment (hfiguiere) [inline org.regular_dev.biplanes_revival.yml:20]:
```suggestion
    post-install:
```

You use build system `cmake-ninja`. So why do you call `ninja`

Comment (Casqade) [reply org.regular_dev.biplanes_revival.yml:20]: I thought it was necessary to call ninja explicitly

File: flathub.json (hunk 3 above)
Comment (Casqade) [reply flathub.json:2]: So I should just delete this file to see what happens?

Comment (bbhtt) [reply flathub.json:2]: Yes

Comment (Casqade) [reply flathub.json]: Ok, thanks

**Comment by bbhtt** [comment]
bot, build org.regular_dev.biplanes_revival

**Pull request closed without merging**